src/share/classes/java/lang/Thread.java

```diff
@@ -1670,26 +1670,12 @@ public void setContextClassLoader(ClassLoader cl) {
      * @since 1.5
      */
     public StackTraceElement[] getStackTrace() {
-        boolean slowPath;
-        if (WEA != null) {
-            WispTask task = this.wispTask;
-            if (task == null) {
-                // When we create a thread, coroutine will not be created immediately.
-                // So if we take the stack trace at once, it will NCE.
-                // See: NullStackTrace.java and 6571589, Thread return an array which size is 0.
-                return new StackTraceElement[0];
-            }
-            if (!WEA.runningAsCoroutine(this)) {
```

Comment:
Why do we remove this?

Reply:
we could use dumpthreads to get stack now , so I think maybe these codes are duplicate.